pilot/cmd/pilot-agent/main.go

```diff
@@ -574,6 +588,11 @@ func init() {
 	proxyCmd.PersistentFlags().StringVar(&tlsClientRootCert, "tlsClientRootCert",
 		model.DefaultRootCert, "Absolute path to client root cert file used for istio mTLS")
 
+	proxyCmd.PersistentFlags().StringVar(&k8sServiceAccountJWTPath, "k8sServiceAccountJWTPath", "",
```

Comment:
Can you instead use detection, and use env variables - which are more friendly to Kustomize ( we are trying to move from helm to kustomize ) ? We also want to have a more consistent way to override settings - see Louis document - this kind of CLI is very difficult to use nicely with JSONpath.